Demo/Sources/Assets/Icons.xcassets/Markets 1/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
these seem to be duplicated:

<img width="232" alt="image" src="https://github.com/finn-no/FinniversKit/assets/23578783/e7f3851d-8ab4-46cb-b272-eec349f7c375">

Reply:
now we have a market icons with Finn and Tori icons